runtime/POSIX/klee_init_env.c

```diff
@@ -123,45 +123,71 @@ usage: (klee_init_env) [options] [program arguments]\n\
   while (k < argc) {
     if (__streq(argv[k], "--sym-arg") || __streq(argv[k], "-sym-arg")) {
       const char *msg = "--sym-arg expects an integer argument <max-len>";
-      if (++k == argc)        
-	__emit_error(msg);
-		
+      if (++k == argc)
+        __emit_error(msg);
+
       max_len = __str_to_int(argv[k++], msg);
-      sym_arg_name[3] = '0' + sym_arg_num++;
-      __add_arg(&new_argc, new_argv, 
-                __get_sym_str(max_len, sym_arg_name),
+
+      if (sym_arg_num > 99)
```

Comment:
100 seems arbitrary. We could use the printable ASCII range `'0'` (47) to `'~'` (126) or incl. overflow the full byte up to `'/'` (46).

Reply:
It is arbitrary but sufficient, no?  Using the printable ASCII set is also arbitrary, and I prefer having numbered arguments (arg00, arg01, etc.).  Alternatively, we could impose no limit, but I don't think it's worth another refactoring.  So unless you feel strongly about this, I'll leave it like this.

Reply:
> I prefer having numbered arguments (arg00, arg01, etc.)

But currently it is `arg0`, `arg1`, ..., `arg}`, `arg~`, `arg\x7f`, ... That's why I've suggested the printable range. We can keep it that way but than we should add a comment that this is an arbitrary limit.

Reply:
No worries, let's merge it then.